configuration/items.md

```diff
@@ -391,7 +391,7 @@ On the filesystem, the following icon files are provided by the user:
 Take note, that the Transformation used in the `Livingroom_Light_Connection` Item doesn't effect the needed state specific icons - the icon selection considers "myerror", not the contents of the `error.map` file.
 
 **Number State Matching Rule:**
-For Number Items the equal or next lowest state icon that can be found will be used.
+For Number or Dimmer Items the equal or next lowest state icon that can be found will be used. The default icon will be used for negative numers, or above 100 i.e. the available filename range is icon-0 to icon-99 only.
```

Comment:
I would remove the `or Dimmer` and extract that to a new sentence.
Also please break down to one sentence per line and make `The default..` appear on its own line.
(We do this to keep a nice track of changes per sentence in git.)

Reply:
So my version would look like this:

> For Number Items...
The default...
Please notice that his also is valid for Dimmer Items since valus below 0 or above 100 are undefined for them.

Of course the sentences can be adapted.
This is just how i think it would look organized.

Reply:
```suggestion
For Number Items the equal or next lowest state icon that can be found will be used.
The default icon will be used as fallback for undefined states. 
Take note that a Dimmer Item is only defined for values between 0 to 100 and the default icon is shown outside that range.
```